polling_stations/apps/file_uploads/templates/file_uploads/upload.html

```diff
@@ -27,61 +27,70 @@ <h2>Javascript disabled</h2>
-                    There is more than one upcoming election in your region. Please pick the date this upload is for.
-                    if you have more than one upcoming election, you will need to upload different data exports for each,
-                    or email us if you would like us to reuse the data.
-                </p>
-                <div class="ds-stack-smallest">
-                    {% for election_date in UPCOMING_ELECTION_DATES %}
-                        <label class="ds-field-radio">
-                            <input type="radio" name="election_date" value="{{ election_date }}" required >
-                            <span>{{ election_date }}</span>
-                        </label>
-                    {% endfor %}
+                <legend>Exported data</legend>
+                <p>Please upload the file(s) exported from your EMS. The accepted file types are CSV or TSV.</p>
 
+                <div class="ds-field">
+                    <label for="file0">File 1</label>
+                    <input type="file" id="file0" name="file0" accept=".csv,.tsv"  />
+                    <progress id="progressBar0" value="0" max="100"></progress>
                 </div>
-            </fieldset>
+                <div class="ds-field">
+                    <label for="file1">File 2 (optional)
+                        <small>Only required if you use the 'Democracy Counts' EMS</small>
+                    </label>
 
-        {% else %}
-            <p>Please upload data for the elections taking place on <strong>{{ UPCOMING_ELECTION_DATES.0 }}</strong></p>
-            <input type="hidden" name="election_date" value="{{ UPCOMING_ELECTION_DATES.0 }}">
-        {% endif %}
-
-        <fieldset>
-            <legend>Exported data</legend>
-            <p>Please upload the file(s) exported from your EMS. The accepted file types are CSV or TSV.</p>
-
-            <div class="ds-field">
-                <label for="file0">File 1</label>
-                <input type="file" id="file0" name="file0" accept=".csv,.tsv"  />
-                <progress id="progressBar0" value="0" max="100"></progress>
-            </div>
-            <div class="ds-field">
-                <label for="file1">File 2 (optional)
-                    <small>Only required if you use the 'Democracy Counts' EMS</small>
-                </label>
-
-                <input type="file" id="file1" name="file1" accept=".csv,.tsv" />
-                <progress id="progressBar1" value="0" max="100"></progress>
-            </div>
-        </fieldset>
-        <button type="submit" class="ds-button" id="submit" disabled>Upload</button>
-    </form>
+                    <input type="file" id="file1" name="file1" accept=".csv,.tsv" />
+                    <progress id="progressBar1" value="0" max="100"></progress>
+                </div>
+            </fieldset>
+            <button type="submit" class="ds-button" id="submit" disabled>Upload</button>
+        </form>
+    {%endif%}
```

Comment:
..and
```suggestion
    {% endif %}
```

Reply:
fixed in d630b7ebef6677f53d5a1fcf9ef1e0c2583e2f14